flowise-embed/src/components/Bot.tsx

```diff
@@ -368,6 +379,38 @@ export const Bot = (botProps: BotProps & { class?: string }) => {
     handleSubmit(prompt);
   };
 
+  createEffect(() => {
+    const lastSelectionMessage = messages().findLast((message) => message.type === 'selectionMessage')?.message;
+    const lastUserMessage = messages().findLast((message) => message.type === 'userMessage')?.message;
+
+    if (
+      [messageUtils.NCM_INITIAL_QUESTION, messageUtils.NCM_CONTINUE_QUESTION, messageUtils.NCM_HELP_QUESTION].includes(
+        lastSelectionMessage ? lastSelectionMessage : '',
+      )
+    ) {
```

Comment:
Não saquei o porquê da parte `lastSelectionMessage ? lastSelectionMessage : ''`. Não está verificando especificamente pelo valor do lastSelectionMessage?
Se a questão for validar a existência de valor nele, poderia fazer assim:
```suggestion
    if (
      lastSelectionMessage &&
      [messageUtils.NCM_INITIAL_QUESTION, messageUtils.NCM_CONTINUE_QUESTION, messageUtils.NCM_HELP_QUESTION].includes(lastSelectionMessage)
    ) {

Reply:
Isso acontece pois o Find usado para formar a constante lastSelectionMessage é possivelmente undefined, e para lidar com isso tive de adicionar o ternário.